halo/evmstaking2/keeper/metrics.go

```diff
@@ -0,0 +1,29 @@
+package keeper
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	eventDeliveries = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
this should be a counter I think. Also see [prometheus](https://prometheus.io/docs/practices/naming/) docs on naming (counters should have `total` suffix)